rocksdb_replicator/rockdb_wrapper.cpp

```diff
@@ -0,0 +1,32 @@
+#include "rocksdb_replicator/rocksdb_wrapper.h"
+
+namespace replicator {
+uint64_t RocksDbWrapper::LatestSequenceNumber() { return db_->GetLatestSequenceNumber(); }
+rocksdb::Status RocksDbWrapper::WriteToLeader(const rocksdb::WriteOptions& options,
+                                              rocksdb::WriteBatch* updates) {
+  return db_->Write(options, updates);
+}
+rocksdb::Status RocksDbWrapper::GetUpdatesFromLeader(
+    rocksdb::SequenceNumber seq_number, std::unique_ptr<rocksdb::TransactionLogIterator>* iter) {
+  return db_->GetUpdatesSince(seq_number, iter);
+}
+bool RocksDbWrapper::HandleReplicateResponse(Update update) {
```

Comment:
will `update` be copied? Can we avoid that?

Reply:
I will change it to a pointer